### PR TITLE
ci: firefox command change 🦊, remove legacy tests 🪓

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6350316975).

Also upgrades a handful of packages

### CODE REVIEW

File: src/app/pages/receive/receive-stx.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export function ReceiveStxModal() {
 
   function copyToClipboard() {
     void analytics.track('copy_stx_address_to_clipboard');
```
Comment (pete-watters) [inline src/app/pages/receive/receive-stx.tsx:18]: A bit less exciting but OK 😄

File: src/app/query/bitcoin/ordinals/ordinals-aware-utxo.query.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export function useOrdinalsAwareUtxoQueries(utxos: TaprootUtxo[] | btc.Transacti
         queryKey: makeOrdinalsAwareUtxoQueryKey(txId, txIndex),
         queryFn: () => fetchOrdinalsAwareUtxo(txId, txIndex),
         select: (resp: OrdApiInscriptionTxOutput) =>
```
Comment (pete-watters) [inline src/app/query/bitcoin/ordinals/ordinals-aware-utxo.query.ts:75]: Is this OK? Looks like it could be a prettier error moving the bracket?

**Review by pete-watters — APPROVED** [review]

File: src/app/query/bitcoin/ordinals/ordinals-aware-utxo.query.ts (hunk 2 above)
Comment (kyranjamie) [reply src/app/query/bitcoin/ordinals/ordinals-aware-utxo.query.ts:75]: Prettier 3.0 🆕

File: tests/selectors/home.selectors.ts [hunk 3]
```diff
@@ -10,4 +10,6 @@ export enum HomePageSelectors {
   BalancesTabBtn = 'tab-balances',
   SwapBtn = 'swap-btn',
   FundAccountBtn = 'fund-account-btn',
+  BalancesList = 'balances-list',
+  BtnFundAccount = 'btn-fund-account',
```
Comment (fbwoolf) [inline tests/selectors/home.selectors.ts]: Shouldn't this be `FundAccountBtn` to follow pattern here?

File: tests/specs/profile/profile.spec.ts [hunk 4]
```diff
@@ -25,14 +25,15 @@ test.describe('Profile updating', () => {
...
     testAppPage = await TestAppPage.openDemoPage(context);
     await testAppPage.signIn();
     const accountsPage = await context.waitForEvent('page');
+    await accountsPage.reload();
+    await accountsPage.evaluate(() => console.log((window as any).LeatherProvider));
     await accountsPage.locator('text="Account 1"').click();
     await testAppPage.page.bringToFront();
     await testAppPage.page.click('text=Profile', {
-      timeout: 30000,
+      // timeout: 30000,
```
Comment (fbwoolf) [inline tests/specs/profile/profile.spec.ts]: Remove?

**Review by fbwoolf — APPROVED** [review]
Nice removing all the tests-legacy code 🚀

File: src/app/pages/receive/receive-stx.tsx (hunk 1 above)
Comment (kyranjamie) [reply src/app/pages/receive/receive-stx.tsx:18]: haha, this was to trigger the build, making changes in `src` prior to other fixes. 

Also, though, I'm also of the opinion that exclamation marks shouldn't often be used in interfaces.

File: tests/selectors/home.selectors.ts (hunk 3 above)
Comment (kyranjamie) [reply tests/selectors/home.selectors.ts]: removed entirely was copied from old tests

File: src/app/pages/receive/receive-stx.tsx (hunk 1 above)
Comment (pete-watters) [reply src/app/pages/receive/receive-stx.tsx:18]: 😄 agreed. I read it as shouting at the users

**Pull request merged**